### DISCUSSION TITLE
What's conversation_id and parent_message_id usage

### DISCUSSION THREAD

**Discussion by Harry-Jing** [acheong08/ChatGPT#98] [Q&A]
if i want to continue my talk to chatGPT, what shuold i do

**Comment by Harry-Jing** [comment]
My code is like this
  chatbot = Chatbot(config, conversation_id=temp_convr_id) 
  chatbot.get_chat_response(user_input_text)
But below problem happen
![image](https://user-images.githubusercontent.com/69247286/205999032-4bb9dd1f-7ccb-45f4-a70c-79abbc014c39.png)
Is there something wrong with the way I implemented it?

**Comment by dreamofmt** [comment]
Sorry to be of little help as I don't even understand how to implement it as I would like to know this answer and it just says Developer and lists a snippet of code, I'm obviously not smart enough to understand the basics to do a lot of this stuff yet. How do you specify which conversation is taking place? I only see what is in RevchatGPT.py on my copy but that doesn't look the same as what is posted in the wiki, also if maintaining multiple conversations do you just modify the value each time you have a session and must be loaded beforehand? 

Are you modifying ai_chat.py (I dont have this file anywhere, maybe my version is needing to be upgraded), or just the revchatgpt.py?

Sorry again to be of little assistance, I wish I could help more

**Comment by Harry-Jing** [comment]
Thanks  for your help.
I find same problem in #7

**Comment by acheong08** [comment]
You have to implement it yourself by caching the parent_id and conversation_id and then setting chatbot.parent_id and chatbot.conversation_id to your cached values
